src/Blocks/custom/carousel/carousel.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Template for the Carousel Block.
+ *
+ * @package Unicorns
+ */
+
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+use Unicorns\Manifest\Manifest;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$blockJsClass = Components::checkAttr('blockJsClass', $attributes, $manifest);
+$isLoop = Components::checkAttr('isLoop', $attributes, $manifest);
+$showItems = Components::checkAttr('showItems', $attributes, $manifest);
+
+$carouselClass = Components::classnames([
+	$blockClass,
+	$blockJsClass,
+	'swiper-container',
+]);
+
+?>
+
+<div class="<?php echo esc_attr($carouselClass); ?>" data-swiper-loop="<?php echo esc_attr($isLoop); ?>" data-show-items="<?php echo esc_attr($showItems); ?>">
+
+	<div class="<?php echo esc_attr('swiper-wrapper'); ?>">
+		<?php echo wp_kses_post($innerBlockContent); ?>
+	</div>
+
+	<div class="<?php echo esc_attr($blockClass); ?>__control-container">
```

Comment:
write this as a <?php echo esc_attr("{$blockClass}__control-container") ?>

Reply:
The buttons I'll switch to `a` or maybe even `button`, but I'm not sure if making the control container an `a` element would be a good idea.

Reply:
No sorry my comment wasn't full.

class should be written as 
```
<?php echo esc_attr("{$blockClass}__control-container"); ?>
```